packages/web/pages/technology/[id]/edit.js

```diff
@@ -203,7 +214,10 @@ const TechnologyFormPage = ({ taxonomies, technology }) => {
 
 	return (
 		<ContentContainer bgColor={colors.gray98}>
-			<Protected>
+			<Protected
+				customIsAuthorized={authorizedToEdit}
+				messageContext={authorizedToEdit ? '' : 'techStatus'}
```

Comment:
```suggestion
				messageContext={authorizedToEdit ? '' : 'techStatus'}
```

Can't we have this logic inside of the component? e.g

```
const Protected = ({ authorizedToEdit }) => {
   const messageContext = authorizedToEdit ? '' : 'techStatus'
   return <div>{messageContext}</div>
}
```

Reply:
I dont think its a good idea because we cant customize the message context this way

Reply:
Fair enough 👍